src/Stratis.Bitcoin/Base/BaseFeature.cs

```diff
@@ -278,25 +278,37 @@ public override void Dispose()
             this.logger.LogInformation("Flushing peers...");
             this.flushAddressManagerLoop.Dispose();
 
+            this.logger.LogInformation("peerAddressManager...");
             this.peerAddressManager.Dispose();
 
+            this.logger.LogInformation("partialValidator...");
```

Comment:
Partial validator should probably get disposed inside CM come to think about it

Reply:
I'll check this out.